src/common/components/repositories-list/index.js

```diff
@@ -75,19 +66,19 @@ class RepositoriesList extends Component {
           </div>
         }
         <Table>
-          <THead>
-            <TR>
-              <TH>Name</TH>
-              <TH>Configured</TH>
-              <TH>Registered for publishing</TH>
-              <TH>Latest build</TH>
-            </TR>
-          </THead>
-          <TBody>
+          <Head>
+            <Row>
+              <Header col="30">Name</Header>
+              <Header col="20">Configured</Header>
+              <Header col="20">Registered for publishing</Header>
+              <Header col="30">Latest build</Header>
+            </Row>
+          </Head>
+          <Body>
```

Comment:
I've replaced the Vanilla-style table with a MAAS-style interactive table. I've adjusted the names of the components to match the naming scheme of the classes used by the MAAS theme.